src/pk_compiler.c

```diff
@@ -1880,6 +1894,8 @@ static void emitAssignment(Compiler* compiler, TokenType assignment) {
     case TK_OREQ: emitOpcode(compiler, OP_BIT_OR); break;
     case TK_XOREQ: emitOpcode(compiler, OP_BIT_XOR); break;
     case TK_MODEQ: emitOpcode(compiler, OP_MOD); break;
```

Comment:
Move this `TK_MODEQ` above to match the order in the `Tokens` enums, (I've forgotten to update it after the `%=` PR)